Brixcer_Box/src/main.cpp

```diff
@@ -156,18 +153,18 @@ void receiveData() {
 
 void buttonClick() {
     for (int i = 0; i < KnobCount; i++) {
-        int button = knobs[i].readButtonValue();
-        if (button) {
-            sendData(i, 0, button);
+        int value = knobs[i].readButtonValue();
+        if (value) {
+            sendData(i, 0, value);
         }
     }
 }
 
 void knobPinTrigger() {
     for (int i = 0; i < KnobCount; i++) {
-        int button = knobs[i].readKnobValues();
-        if (button != 0) {
-            sendData(i, 0, button);
-        } 
```

Comment:
Same as above

Reply:
This one in particular is because the variable was used to describe the value from the knob, not any button press. Then the one above came from making them look alike. Besides, button makes more sense as describing the pin the button is on rather than the value read from the pin